stable/mlrun-kit/templates/mlrun-persistency/pvc-api.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.mlrun.api.persistence.enabled -}}
```

Comment:
put under /stable/mlrun-kit/templates/persistency folder and named
mlrun-api-pvc.yaml

(see comment above)